Services/TokenAuthentication/ModsenOnlineStore.Login.Application/Services/LoginService.cs

```diff
@@ -74,6 +74,25 @@ public async Task<DataResponseInfo<User>> GetUserByIdAsync(int id)
             return new DataResponseInfo<User>(data: user, success: true, message: $"user with id {user.Id}");
         }
 
+        public async Task<ResponseInfo> MakePaymentAsync(int id, decimal money)
```

Comment:
I think this method is not responsibility of login service

Reply:
Should I create a separate payment service (user microservice)? Is it worth not moving the functions of getting information about the user into a new user microservice?

Reply:
done